manticore/native/state.py

```diff
@@ -0,0 +1,44 @@
+from manticore.core.state import StateBase, Concretize, TerminateState
+from manticore.native.memory import ConcretizeMemory, MemoryException
+
+
+class State(StateBase):
```

Comment:
i wonder if we should not use the same name `State` for both ethereum and native. `NativeState` and `EthereumState`?

Reply:
@yan what do you think here?

I am up for using `NativeState` and `EthereumState`, being more explicit is probably better here.

Reply:
I don't have strong opinions on this. I would actually have been fine with `native.State` and `evm.State`, since module/namespace can describe the kind, not the name itself, but `NativeState` and `EthereumState` are both fine with me.